imports/plugins/core/dashboard/client/components/plugin-versions.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+
+export default class PluginVersion extends Component {
```

Comment:
In new code we name the file the same as the component, in the same case: `PluginVersion.js`